src/BookSettings.ts

```diff
@@ -34,20 +34,23 @@ export default class BookSettings {
     private static readonly SELECTED_VIEW_KEY = "settings-selected-view";
     private static readonly SELECTED_FONT_SIZE_KEY = "settings-selected-font-size";
 
+    /** @param store Store to save the user's selections in. */
+    /** @param bookViews Array of BookView options. */
     /** @param fontSizes Array of font size strings sorted from smallest to largest. */
-    public static async create(bookViews: BookView[], fontSizes: string[], store: Store) {
-        const settings = new this(bookViews, fontSizes, store);
-        await settings.initializeSelections();
+    /** @param defaultFontSize Initial font size to use until the user makes a selection. */
+    public static async create(store: Store, bookViews: BookView[], fontSizes: string[], defaultFontSize?: string) {
```

Comment:
It might be better to just take a `fontSizesinPixels: [number]` and do `fontSizesInPixels.map(n => n + "px")` internally so people don't accidentally pass font sizes like "100%" or "14pt".